examples/webgl_multiple_elements_with_text.html

```diff
@@ -0,0 +1,429 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - multiple elements with text</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			* {
+				box-sizing: border-box;
+				-moz-box-sizing: border-box;
+			}
+
+			body {
+				color: #000;
+				font-family: Monospace;
+				font-size: 13px;
+
+				background-color: #fff;
+				margin: auto;
+				padding: .5in;
+				max-width: 7in;
+				text-align: justify;
+			}
+
+			.view {
+				width: 5in;
+				height: 5in;
+				margin: auto;
+			}
+
+			#c {
+				position: fixed;
+				left: 0px; top: 0px;
+				width: 100%;
+				height: 100%;
+				background-color: #fff;
+				z-index: -1;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 6.5in;
+				padding: 0px;
+				text-align: center;
+			}
+
+			a {
+				color: #0080ff;
+			}
+
+		</style>
+	</head>
+	<body>
+
+		<canvas id="c"></canvas>
+
+		<div id="info"><a href="http://threejs.org" target="_blank">three.js</a> - multiple elements with text - webgl</div>
+
+		<script src="../build/three.min.js"></script>
+		<script src="../examples/js/controls/OrbitControls.js"></script>
+
+		<script src="js/Detector.js"></script>
+		
+		<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
```

Comment:
The point of the example is to display math in close proximity to its visualization, both in the code and rendered, so that one can see the connection between the two. To that end the math is necessary.

Perhaps I should have named the example "webgl_multiple_elements_with_math". I can change it if you want.